kernel-r/declarativewidgets/R/serializers.r

```diff
@@ -1,5 +1,14 @@
 #' @include serializer.r
 
+serialize_element <- function(elem) {
+    if(class(elem) == "Date") {
+        options(digits.secs=3)
+        return (format(elem, "%Y-%m-%dT%H:%M:%OSZ"))
```

Comment:
pandas supports 'iso' or 'epoch' (milliseconds) as possible date_time formats.  Notebook users can pass in either one.  Perhaps we should attempt to support the same behavior for R and Spark DataFrames too?